dev/TEAL.md

```diff
@@ -440,7 +449,7 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | 16 | TypeEnum | uint64 |      | See table below |
 | 17 | XferAsset | uint64 |      | Asset ID |
 | 18 | AssetAmount | uint64 |      | value in Asset's units |
-| 19 | AssetSender | []byte |      | 32 byte address. Causes clawback of all value of asset from AssetSender if Sender is the Clawback address of the asset. |
+| 19 | AssetSender | []byte |      | 32 byte address. Moves asset from AssetSender if Sender is the Clawback address of the asset. |
```

Comment:
```suggestion
| 19 | AssetSender | []byte |      | 32 byte address. Used to move asset from AssetSender if Sender is the Clawback address of the asset. |
```

Reply:
Agree that it should be reworded to describe the field, rather than sounding like the field is a command.  How about:
"32 byte address. The source of assets if Sender is the Asset's Clawback address."

The change to "Asset's Clawback address" at the end is to match the syntax of "AssetAmount", which reads:
"value in Asset's units"

Maybe "value" that should be capitalized for AssetAmount.

Reply:
"32 byte address. The source of assets if Sender is the Asset's Clawback address."

looks good